src/components/map-marker/readme.md

```diff
@@ -0,0 +1,28 @@
+# gx-map-marker
+
+This web component must be included inner _GX Map_ Component (inner `<gx-map>`).
+`coords` attribute must always be included in the the tag. If you do not set any attribute, the marker will initialize with default values and it will be draw in `0, 0` position of the map.
```

Comment:
```suggestion
`coords` attribute must always have a value. If a value is not set, the marker will be initialized with default values and it will be drawn at `0, 0` coordinates.
```

Reply:
Ready!